File: port_scan.php
<?php
//set_time_limit(0);
$target_ip = '127.0.0.1';
$ports = [80,81,88,6379,8000,8080,8088];
$timeout = 0.5;
foreach ($ports as $p) {
    # code...
    $f = @fsockopen($target_ip,$p,$errno, $errstr, $timeout);
    if(is_resource($f)){
        echo 'port : '.$p.'=>open<br>';
        fclose($f);
    }
}


?>

File: port_scan.py
import socket
def scan(port):
  s = socket.socket()
  if s.connect_ex(('localhost', port)) == 0:
      print port, 'open'
      s.close()
if __name__ == '__main__':
    ports = [80,81,88,6379,8000,8080,8088]
    map(scan,ports)
